clang/runtime/dpct-rt/include/dpct/device.hpp

```diff
@@ -582,6 +582,19 @@ class device_ext : public sycl::device {
     lock.lock();
   }
 
+  std::vector<sycl::event> get_last_events() {
```

Comment:
Consider add a static function for current device,  just as get_default_queue();